File: mvp-java/src/main/java/com/wf/mvp/java/presenter/DefaultIPresenter.java
package com.wf.mvp.java.presenter;

import android.os.Message;

/**
 * MvpMode -> com.wf.mvp.java.presenter -> DefaultPresenter
 *
 * @Author: wf-pc
 * @Date: 2020-05-09 21:08
 * <p>
 * The default Presenter is to satisfy the situation where Presenter is not needed.
 */
public class DefaultIPresenter extends IPresenter {

    @Override
    protected void init() {

    }

    @Override
    protected void release() {

    }

    @Override
    public void onHandleMessage(Message msg) {

    }

}

File: mvp-kotlin/src/main/java/com/wf/mvp/kotlin/presenter/DefaultIPresenter.kt
package com.wf.mvp.kotlin.presenter

import android.os.Message
import com.wf.mvp.kotlin.impl.Presenter
import com.wf.mvp.kotlin.impl.Ui

/**
 * MvpMode -> com.wf.mvp.kotlin.presenter -> DefaultPresenter
 * @Author: wf-pc
 * @Date: 2020-05-10 16:51
 * <p>
 * The default Presenter is to satisfy the situation where Presenter is not needed.
 */
class DefaultIPresenter : IPresenter<Ui>() {

    override fun init() {
    }

    override fun release() {
    }

    override fun onHandleMessage(msg: Message) {
    }

}
